pkg/api/controllers/account_controller_test.go

```diff
@@ -316,6 +316,19 @@ func TestGetAccounts(t *testing.T) {
 					assert.Equal(t, cursor.Data[0].Address, "bob")
 				})
 
+				// test filter by balance != 100
```

Comment:
this comment seems a bit useless to me, as the following sub test name already tells what it is testing :)

Reply:
you're right ! i'll be careful not to add those kind of comments again in next PRs :)